src/libraries/System.Text.Json/src/System/Text/Json/Serialization/Converters/Object/ObjectDefaultConverter.cs

```diff
@@ -48,10 +49,10 @@ internal override bool OnTryRead(ref Utf8JsonReader reader, Type typeToConvert,
                     // Read method would have thrown if otherwise.
                     Debug.Assert(tokenType == JsonTokenType.PropertyName);
 
+                    ReadOnlySpan<byte> unescapedPropertyName = JsonSerializer.GetPropertyName(ref state, ref reader, options);
```

Comment:
Note that `GetPropertyName` is marked with `AggressiveInline` but `LookupProperty` is not (it used to be, but there became 4 references to it over time causing code bloat).